### PR TITLE
[manager] Add zero-out-dram to config_runtime.ini; disable by default

### PR DESCRIPTION
**Pull request by davidbiancolin**

### CODE REVIEW

File: deploy/sample-backup-configs/sample_config_runtime.ini [hunk 1]
```diff
@@ -48,3 +48,9 @@ readrate=0
 [workload]
 workloadname=linux-uniform.json
 terminateoncompletion=no
+
+[hostdebug]
+# When enabled (=yes), Zeros-out FPGA-attached DRAM (takes 2-5 minutes)
```
Comment (sagark) [inline deploy/sample-backup-configs/sample_config_runtime.ini]:
```suggestion
# When enabled (=yes), Zeros-out FPGA-attached DRAM before simulations
# begin (takes 2-5 minutes).
```

**Comment by sagark** [comment]
@davidbiancolin actually i just realized, can you update the docs too?

**Comment by davidbiancolin** [comment]
> @davidbiancolin actually i just realized, can you update the docs too?

Yeah, i'll do that. This is just adding a blurb to the runtime_config.ini docs?

**Comment by sagark** [comment]
> > @davidbiancolin actually i just realized, can you update the docs too?
> 
> Yeah, i'll do that. This is just adding a blurb to the runtime_config.ini docs?

Yeah

**Pull request merged**